### DISCUSSION TITLE
Having multiple data types in same column

### DISCUSSION THREAD

**Discussion by turkerdev** [prisma/prisma#9135] [Q&A] [answered]
i wonder if there is another better way to achieve something like this, i mean having multiple data types in same column.

```prisma
model Sinema {
  IMDBID String     @id
  type   SinemaType
  Series Series?
  Movie  Movie?
}

enum SinemaType {
  Movie
  Series
  Episode
  Anime
  AnimeEpisode
}

model Series {
  slug         String @unique
  title        String
  cinema       Sinema @relation(fields: [cinemaIMDBID], references: [IMDBID])
  cinemaIMDBID String
}

model Movie {
  slug         String  @unique
  title        String
  released     String
  genre        Genre[]
  image        String
  imdbRating   String
  cinema       Sinema  @relation(fields: [cinemaIMDBID], references: [IMDBID])
  cinemaIMDBID String
}

enum Genre {
  Action
  Adventure
  Animation
//...
}
```

**Comment by ryands17** [comment, marked as answer]
@bajosiaa 👋 

Not at the moment, but we do have a feature request for this that you can add a 👍 to so that we know the priority for this: https://github.com/prisma/prisma/issues/2505

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
